src/backend/joanie/core/models/courses.py

```diff
@@ -160,6 +160,13 @@ class Organization(parler_models.TranslatableModel, BaseModel):
     )
     signature = models.ImageField(_("signature"), blank=True)
     logo = ThumbnailerImageField(_("logo"), blank=True)
+    address = models.OneToOneField(
+        to=Address,
+        verbose_name=_("address"),
+        on_delete=models.RESTRICT,
+        blank=True,
+        null=True,
+    )
```

Comment:
There is no issues to create an address if user is not set ?

Reply:
I didn't see any, but maybe I missed it.

Reply:
In fact you are assigning to Address an owner.
But this relation currently cannot be null.

```python
django.core.exceptions.ValidationError: {'owner': ['This field cannot be null.']}
```

Reply:
As we discussed, the address relation has been removed.